permissions/plugin-opsgenie.yml

```diff
@@ -0,0 +1,7 @@
+---
+name: "opsgenie"
+paths:
+- "com/opsgenie/integration/opsgenie"
+- "org/jenkins-ci/plugins/opsgenie"
```

Comment:
This seems to be unused. Are there plans to use this path or why is it included?